pkg/handler/tool_handler.go

```diff
@@ -84,38 +82,21 @@ func (h *ToolHandler) Cleanup(ctx context.Context, r *v1alpha1.ToolRequest) erro
 
 func (h *ToolHandler) run(tool string, cmds []string) error {
 	for i, c := range cmds {
-		args := splitArgs(c)
-		cmd := exec.Command(tool, args...)
+		script := fmt.Sprintf("%s %s", tool, c)
+		cmd := exec.Command("bash", "-c", script)
```

Comment:
Comment here and in the CLI desc that tool only works where bash is available.

Reply:
Also comment here about why we use bash.

Reply:
ack